catroid/src/androidTest/java/org.catrobat.catroid/uiespresso/util/matchers/AdapterViewMatchers.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2017 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.catroid.uiespresso.util.matchers;
+
+import android.view.View;
+import android.widget.Adapter;
+import android.widget.AdapterView;
+
+import org.catrobat.catroid.common.ProjectData;
+import org.catrobat.catroid.ui.adapter.ProjectListAdapter;
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+
+public final class AdapterViewMatchers {
+	private AdapterViewMatchers() {
+		throw new AssertionError();
+	}
+
+	public static Matcher<View> withAdaptedDataAt(final String expectedContent, final int listPosition) {
```

Comment:
calling a view matcher withAdapted**Data**At is not a good idea, since in espresso we got view- and data- Matchers/Actions/etc, and those are different things